CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: The option of "Remove Bars in Great Tree Hall". This feature removes the Boost Ball bars obstacle in Tallon Overworld's Great Tree Hall allowing free movement between the lower and upper levels of the room.
 - Changed: Damage Reduction for Starter Preset and Moderate Challenge is now Additive.
 - Fixed: Setting Screen Brightness in the cosmetic settings mismatching with what appears in-game.
+- Changed: Vanilla Missile Blast Shields have been changed to the Randomprime Missile Blast Shields. The corresponding doors were also changed to add the needed Radomprime Missile Blast Shield on the other side.
```

Comment:
"randomprime blast shields" is jargon for an end user. please rephrase this. quick ideas
- always two-way
- auto-open the door when broken
- destroy themselves when opened from behind
- door cannot be shot underneath the shield

Reply:
Changed the wording, please let me know if it is now better